docs/sources/setup-grafana/installation/mac/index.md

```diff
@@ -29,44 +31,36 @@ Use [Homebrew](http://brew.sh/) to install the most recent released version of G
    - `/usr/local/Cellar/grafana/[version]` (Homebrew v2)
```

Comment:
@DanCech , Are these homebrew versions still valid? Are there others we should list?

Reply:
The current homebrew version is v4, and the install path actually seems to depend on the hardware itself, `/usr/local/` for Intel silicon and `/opt/homebrew/` for Apple silicon.
```suggestion
   - `/usr/local/Cellar/grafana/[version]` (Intel Silicon)
```